components/camel-tracing/src/main/java/org/apache/camel/tracing/SpanAdapter.java

```diff
@@ -23,24 +23,12 @@ public interface SpanAdapter {
 
     void setError(boolean error);
 
-    void setTag(Tag key, String value);
-
-    void setTag(Tag key, Number value);
-
     void setTag(String key, String value);
 
     void setTag(String key, Number value);
 
     void setTag(String key, Boolean value);
 
-    default void setLowCardinalityTag(Tag key, String value) {
```

Comment:
These are breaking changes, are we ok with that?

Reply:
Yeah Camel end users dont really work directly with this API. And we have migration docs where we can write the steps people need to migrate. And its better to get ahead of this early in Camel v4 than later.

Reply:
If that's fine for you that's fine for me too 😃